### PR TITLE
Collect IPv6 Networking Information in Log Collector Script

### PR DESCRIPTION
**Pull request by jdn5126**
**Issue #, if available:**
https://github.com/awslabs/amazon-eks-ami/issues/1474

**Description of changes:**
This PR updates the log collector script for Linux nodes to collect IPv6 networking information that will assist in debugging. The following commands are run on IPv4 and IPv6 nodes:
1. `ip6tables --wait 1 --numeric --verbose --list --table mangle`
2. `ip6tables --wait 1 --numeric --verbose --list --table filter`
3. `ip6tables --wait 1 --numeric --verbose --list --table nat`
4. `ip6tables --wait 1 --numeric --verbose --list`
5. `ip6tables-save`
6. `timeout 75 conntrack -L -f ipv6`
7. `timeout 75 ip -6 rule show`
8. `timeout 75 ip -6 route show table all`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Testing Done**

I ran the updated script against IPv4 and IPv6 Linux nodes and validated that the expected information was collected.

*[See this guide for recommended testing for PRs.](../doc/CONTRIBUTING.md#testing-changes) Some tests may not apply. Completing tests and providing additional validation steps are not required, but it is recommended and may reduce review time and time to merge.*

### CODE REVIEW

File: log-collector-script/linux/eks-log-collector.sh [hunk 1]
```diff
@@ -509,6 +515,18 @@ get_sysctls_info() {
   ok
 }
 
+get_network_policy_ebpf_info() {
+  try "collect network policy ebpf loaded data"
+  echo "*** EBPF loaded data ***" >> "${COLLECT_DIR}"/networking/ebpf-data.txt
+  LOADED_EBPF=$(/opt/cni/bin/aws-eks-na-cli ebpf loaded-ebpfdata | tee -a "${COLLECT_DIR}"/networking/ebpf-data.txt)
```
Comment (orsenthil) [inline log-collector-script/linux/eks-log-collector.sh:521]: Should we test for the presence of this binary `aws-eks-na-cli` before using it?

Comment (orsenthil) [reply log-collector-script/linux/eks-log-collector.sh:521]: I see the related discussion here https://github.com/awslabs/amazon-eks-ami/pull/1595/files

Comment (orsenthil) [reply log-collector-script/linux/eks-log-collector.sh:521]: Sine this is already merged, not sure why this is here in this diff.

**Review by orsenthil — APPROVED** [review]
LGTM

File: .github/workflows/sync-to-codecommit.yaml [hunk 2]
```diff
@@ -27,5 +27,5 @@ jobs:
       - run: git remote add codecommit ${{ secrets.AWS_CODECOMMIT_REPO_URL }}
       - run: git checkout master
       - run: git push codecommit master
-      - run: git checkout al2023
-      - run: git push codecommit al2023
+      - run: git checkout main
```
Comment (jaydeokar) [inline .github/workflows/sync-to-codecommit.yaml:30]: Probably need to revert the unintentional changes in this PR

Comment (jdn5126) [reply .github/workflows/sync-to-codecommit.yaml:30]: These came from Carter committing to this branch. Not sure what is happening there, so I will reach out to him

File: log-collector-script/linux/eks-log-collector.sh (hunk 1 above)
Comment (jdn5126) [reply log-collector-script/linux/eks-log-collector.sh:521]: Looks like this came from either Carter committing directly to this branch or fallout from the name of the primary branch being changed to `main`. Either way, I will delete and recreate this PR to fix it

**Pull request closed without merging**